xml/Sector.xml

```diff
@@ -249,6 +249,18 @@
             </description>
             <category>121</category>
         </codelist-item>
+        <codelist-item status="withdrawn" activation-date="2019-01-01">
```

Comment:
@akmiller01  this looks like another one where codelist has been listed as "withdrawn", while it should be "active". Maybe there was an error in the code replacing "Vonlontary basis" with "withdrawn" which is incorrect?
![image](https://user-images.githubusercontent.com/24607542/76516750-e7aa9200-6453-11ea-8280-97aa7648ebe9.png)

Reply:
Yep, I was recoding "Vonlontary basis" as withdrawn. This will be corrected in the next commit.

Reply:
Thanks @akmiller01 !